package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-cli-update",
-  "version": "0.9.0",
+  "version": "0.9.1",
```

Comment:
You might need to rebase to get rid of this.

Reply:
Oh, whoops, sure, when this is ready to go otherwise I'll rebase it into oblivion 😄